README.md

```diff
@@ -35,9 +35,17 @@ storageClasses:
     name: "<name>"
     metaurl: "<redis-url>"
     storage: "<storage-type>"
-    access-key: "<access-key>"
-    secret-key: "<secret-key>"
+    accessKey: "<access-key>"
```

Comment:
Should we keep these backward compatible?

Reply:
The keys are `accessKey` and `secretKey` in values.yaml and I modified them wrong in https://github.com/juicedata/juicefs-csi-driver/pull/101, and fix it here 😂.